contents/swift/welcome-to-swift.md

```diff
@@ -126,24 +174,67 @@ Next, we introduce the structs that conform to the protocols above.
 ```swift
 // Penguins can't fly ):
 struct Penguin: Bird {
-    let name: String
-    let canFly = false
+    var canFly: Bool {
+        return false
+    }
 }
 
 struct Eagle: Bird, Flyable {
-    let name: String
-    let canFly = true
+    var canFly: Bool {
+        return true
+    }
  
     var airspeedVelocity: Double {
         return 160.0
-  }
+    }
 }
 ```
 
 And if you haven't noticed, protocols are extremely similar to interfaces in Java. 
 
 To understand more about POP, watching this [WWDC 2015 talk](https://www.youtube.com/watch?v=g2LwFZatfTI) is highly recommended.
 
+## Extensions
+
+Extensions add new functionality to an existing class, structure, enumeration, or protocol type. Using the example of `Eagle` above, we can segment the code neatly by using extensions to implement each protocol separately:
```

Comment:
Perhaps, you can add some method examples (`func`)?

Can consider adding:
It is not possible to add stored property in extensions.
Another separate section on computed properties?

Reply:
> Another separate section on computed properties?

I'll just put a short note on this and refer the user to the Swift docs? :P